FAQ.md

```diff
@@ -121,8 +121,7 @@ accessible and user workloads would be evicted and rescheduled as nodes are upda
 
 ## Should I use Fedora CoreOS 32 or 33 for installing on User Provisioned Infrastructure?
 
-There is currently an issue with using Fedora CoreOS 33 for UPI installations. Please use the FCOS 32 image. After installation, the nodes will have rebooted and updated to the latest stable FCOS 33. 
-
+There is currently an issue with using Fedora CoreOS 33 for UPI installations. Please use the FCOS 32 image. After installation, the nodes will have rebooted and updated to the latest stable FCOS 33. For example, you can use the [FCOS 32 .ova image](https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20201104.3.0/x86_64/fedora-coreos-32.20201104.3.0-vmware.x86_64.ova) for vSphere installations. 
```

Comment:
I don't think its still actual. F33 as a base should work fine.

Lets remove this and then re-add if folks complain?

Reply:
Done.